integration-tests/README.md

```diff
@@ -1,33 +1,109 @@
 
-You can also use more variables - see the table below:
-| Environment Variable | Description | Default value | Example |
+Find the supported variables in the table below:
+| Variable name | Description | Required | Default Value |
+| -- | -- | -- | -- |
+| `HAC_BASE_URL` | The URL to the main page of RHTAP UI | Yes | 'https://prod.foo.redhat.com:1337/beta/hac/application-pipeline' |
+| `USERNAME` | Username for SSO login | Yes | '' |
+| `PASSWORD` | Password for SSO login | Yes | '' |
+| `PR_CHECK` | Assume the test is a PR check, enable report portal, assume keycloak is used instead of RH-SSO | For PR checks | false |
+| `PERIODIC_RUN` | Assume the test is a nightly run, keycloak is used instead of RH-SSO | For runs using ephemeral RHTAP environment | false |
+| `REMOVE_APP_ON_FAIL` | Clean up applications from the cluster even if tests fail | No | false |
+| `GH_TOKEN` | Github token for network requests | For the `advanced` spec | '' |
 | -- | -- | -- | -- |
-| `HAC_BASE_URL` | Base URL for testing | 'https://prod.foo.redhat.com:1337/beta/hac/application-pipeline' | HAC_BASE_URL=https://prod.foo.redhat.com:1337/hac/application-pipeline |
-| `USERNAME` | Username for testing (SSO) | '' |  USERNAME=admin |
-| `PASSWORD` | Password for testing (SSO) | '' | PASSWORD=adminPassword |
-| `KUBECONFIG` | Kubeconfig for cleaning namespace | '/home/user/.kube/appstudio-config' | KUBECONFIG=/home/user/kube/config |
-| `CLEAN_NAMESPACE` | Clean namespace | 'false' | CLEAN_NAMESPACE = 'true' |
+| `CYPRESS_RP_TOKEN` | Report portal token for PR checks | No | '' |
+| `CYPRESS_GH_PR_TITLE` | Report portal setup | No | '' |
+| `CYPRESS_GH_PR_LINK` | Report portal setup | No | '' |
+| `GH_COMMENTBODY` | Report portal setup | No | '' |
+
+### Running from source
+This is the recommended way when either developing tests, or a headed test runner is preferred. After running `npm install` and setting the appropriate environment variables, use one of the following commands to run cypress.
+
+For the headed test runner:
+```
+$ npx cypress open
+```
+Select E2E testing and the browser of your choice (chrome is recommended though), then launch any spec by clicking it in the list, and watch as it runs.
+
+For headless execution:
+```
+$ npx cypress run
+```
+Runs all available specs, by default in the Electron browser. Flags that might come in handy are `-b` to specify the browser, or `-s` to filter spec files based on a glob pattern. For example, running the basic happy path spec in chrome:
+```
+$ npx cypress run -b chrome -s 'tests/basic-happy-path*'
+```
 
-## Docker image used for tests
-To do a changes in Docker image that we're using, update the Dockerfile in `/docker` folder. Once changes are done, build your new image from that folder:
+### Running using docker image
```

Comment:
I don't see the info about where to find the artifacts (screen/video). Do we want to have it there?